lib/av_client/crypto/util.ts

```diff
@@ -0,0 +1,75 @@
+import * as crypto from "../aion_crypto";
+import * as sjcl from "../sjcl";
+
+// As this is working with untyped SJCL classes,
+// we need the _any_ type in this wrapper.
+/*eslint-disable @typescript-eslint/no-explicit-any*/
+
+
+export const Curve = crypto.Curve;
+
+// Converter functions
+// --------------------------
+export const pointFromBits = (bits) => crypto.pointFromBits(bits);
+
+export const pointFromHex = (hex: string): Point => new Point(pointFromBits(sjcl.codec.hex.toBits(hex)));
+export const pointToHex = (point: Point): string => sjcl.codec.hex.fromBits(point.toBits(true));
+
+export const bignumFromHex = (hex: string): Bignum => new Bignum(sjcl.bn.fromBits(sjcl.codec.hex.toBits(hex)));
+export const bignumToHex = (bignum: Bignum): string => sjcl.codec.hex.fromBits(bignum.toBits());
+
+export const hashToBignum = (hash: BitArray): Bignum => new Bignum(crypto.hashToBn(hash));
+
+// Other
+// --------------------------
+export const generateRandomBignum = () => new Bignum(crypto.randomBN());
+
+export const pointFromX = (x: Bignum): Point => {
+  const flag = !x.isEven() ? 2 : 3;
+  const flagBignum = new sjcl.bn(flag);
+
+  const encodedPoint = sjcl.bitArray.concat(flagBignum.toBits(), x.toBits());
+
+  return new Point(pointFromBits(encodedPoint));
+}
+
+export const addPoints = (a: Point, b: Point): Point => {
+  return new Point(crypto.addPoints(a.toEccPoint(), b.toEccPoint()));
+}
+
+// Types
+// --------------------------
+export class Bignum {
```

Comment:
I would move this to a separate file and add tests.

Reply:
Done